plugin/src/main/java/org/opennms/velocloud/requisition/CustomerRequisitionProvider.java

```diff
@@ -81,4 +226,7 @@ public void setEnterpriseId(final UUID enterpriseId) {
         }
     }
 
+    private InetAddress getInetAddress(String ip) {
```

Comment:
I'll switch to using the Util method once the other PR is merged

Reply:
What PR is this talking about?

Reply:
[This](https://github.com/OpenNMS/opennms-velocloud-plugin/pull/11/files#diff-cb63cf1591b1dd5f454cda6acc2c87bf84544753f0d66637f26d6ff195c672fdR34) method from Alberto's PR.